error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/util/ThirdPartyLibraryTest.java

```diff
@@ -18,12 +18,9 @@
 import reactor.core.publisher.Flux;
 
 final class ThirdPartyLibraryTest {
-  private final CompilationTestHelper compilationTestHelper =
-      CompilationTestHelper.newInstance(TestChecker.class, getClass());
```

Comment:
There's also a `BugCheckerRefactoringTestHelper` field in `SourceCodeTest`, but that's part of #561.